src/jobs/time-tracking-job.js

```diff
@@ -11,30 +11,48 @@
  *
  */
 
+const moment = require('moment');
+
 const BaseJobHandler = require('./base/base-job-handler');
+const FogAccessTokenService = require('../services/iofog-access-token-service');
+const logger = require('../logger');
 const Tracking = require('../tracking');
 const TrackingEventType = require('../enums/tracking-event-type');
+const TransactionDecorator = require('../decorators/transaction-decorator');
+
+
+const INTERVAL = 5 * 60 * 1000;
 
 class TimeTrackingJob extends BaseJobHandler {
 
   constructor() {
     super();
-    this.scheduleTime = intervalMin * 60 * 1000;
+    this.startTime = moment.now();
   }
 
   run() {
-    setInterval(trackTime, this.scheduleTime);
```

Comment:
this.scheduleTime time is field of BaseJobHandler and using it is a common way to work with jobs. I think that we should use it here to keep a same style. What do you think?